src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentRenderModeDirectiveIntegrationTests.cs

```diff
@@ -236,4 +236,159 @@ @rendermode TestComponent.myRenderMode
         var assemblyResult = CompileToAssembly(compilationResult, throwOnFailure: false);
         assemblyResult.Diagnostics.Verify();
     }
+
+    [Fact]
+    public void RenderMode_With_SimpleExpression()
```

Comment:
Do we not have any tests that show the compiled code?

Reply:
Added some integration tests. They're basically the same as the identifier form in the code they emit, so not particularly interesting.